commands/utils/utils.go

```diff
@@ -23,3 +28,25 @@ func Chdir(dir string) (cbk func(), err error) {
 	}
 	return func() { err = os.Chdir(wd) }, err
 }
+
+func ReportUsage(commandName string, serverDetails *config.ServerDetails, usageReportSent chan<- error) {
+	var err error
+	defer func() {
+		// Signal usage report finished
```

Comment:
// Signal usage report finished
-->
// The usage reporting is meant to run asynchronously, so that the actual action ins't delayed.
// It is however important to the application to not exit before the reporting is finished. That is, in case the reporting takes lomger than the action.